src/sentry/http.py

```diff
@@ -185,7 +185,7 @@ def safe_urlopen(
     headers=None,
     allow_redirects=False,
     timeout=30,
-    verify_ssl=True,
+    verify_ssl=False,
```

Comment:
these are  in here temporarily because the github enterprise test instance doesn't have a valid cert

Reply:
👀 

This should absolutely be configurable in the integration. It's likely that more than 0 GH enterprise installs use an non-standard cert. Whether it's an internally trusted cert, self signed, something that we can't validate.

Reply:
Yeah, it will have to be a toggle on in the configuration form.